pkg/plugin/standalone/v1/internal/fromproto/specifier.go

```diff
@@ -58,7 +58,7 @@ func SpecifierSpecifyResponse(in *connectorv1.Specifier_Specify_Response) (plugi
 
 func SpecifierParameter(in *connectorv1.Specifier_Parameter) (plugin.Parameter, error) {
 	var validations []plugin.Validation
-	if in.Required {
+	if in.Required { //nolint:staticcheck // we use it to be backwards compatible
```

Comment:
This is here just to make the linter succeed on [this PR](https://github.com/ConduitIO/conduit/pull/751/files#diff-54c07ca8d7960982031b7ee7c488d0061cd54ff8c11d7b801b08032a1572fdf3R91), it will cause a conflict with the PR that properly addresses this. We will address this when merging to main.

Reply:
PR rebased and conflict addressed.